website/pages/home/HomePage.module.css

```diff
@@ -1,4 +1,6 @@
 .homePage {
+  overflow-x: hidden;
```

Comment:
This breaks the terminal animation on the homepage

Reply:
Argh, good catch - can't believe I missed that.